src/Search/SearchEngine.php

```diff
@@ -128,6 +128,15 @@ public static function getMetaItemtypeAvailable($itemtype): array
                 continue;
             }
 
+            if ($key === 'itil_types') {
+                $timeline_types = [\ITILFollowup::class, \CommonITILTask::class, \ITILSolution::class, \CommonITILValidation::class];
+                foreach ($timeline_types as $timeline_type) {
+                    if (is_a($item, $timeline_type) || is_subclass_of($item, $timeline_type)) {
```

Comment:
`is_a()` is already testing the whole inheritance chain.
```suggestion
                    if (is_a($item, $timeline_type)) {
```